softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/factories/ResponseFactory.java

```diff
@@ -0,0 +1,26 @@
+package com.softgroup.common.protocol.factories;
+
+import com.softgroup.common.protocol.*;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 29.03.17.
+ */
+
+public class ResponseFactory {
+
+    public static <T extends ResponseData> Response<T> createResponse(ActionHeader header, T data){
+        return new Response.ResponseBuilder<T>()
+                .setHeader(header)
+                .setData(data)
+                .setStatus(new ResponseStatus(200, "ok"))
```

Comment:
@Dominys Because Response has field ResponseStatus, enum ResponseStatusType is separate class